examples/CampaignManagement/AddCompleteCampaignsUsingMutateJob.cs

```diff
@@ -0,0 +1,579 @@
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            codeExample.Run(new GoogleAdsClient(), customerId);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example adds complete campaigns including campaign budgets, " +
+                    "campaigns, ad groups and keywords using MutateJobService.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
+        public void Run(GoogleAdsClient client, long customerId)
+        {
+            // Get the CampaignBidModifierService.
+            MutateJobServiceClient mutateJobService =
+                client.GetService(Services.V3.MutateJobService);
+
+            // Send the operation in a mutate request.
+            try
+            {
+                string mutateJobResourceName = CreateMutateJob(mutateJobService, customerId);
+                AddAllMutateJobOperations(mutateJobService, customerId, mutateJobResourceName);
+                Operation<Empty, MutateJobMetadata> operationResponse =
+                    RunMutateJob(mutateJobService, mutateJobResourceName);
+                PollMutateJob(operationResponse);
+                FetchAndPrintResults(mutateJobService, mutateJobResourceName);
+            }
+            catch (GoogleAdsException e)
+            {
+                Console.WriteLine("Failure:");
+                Console.WriteLine($"Message: {e.Message}");
+                Console.WriteLine($"Failure: {e.Failure}");
+                Console.WriteLine($"Request ID: {e.RequestId}");
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Creates the mutate job.
+        /// </summary>
+        /// <param name="mutateJobService">The mutate job service.</param>
+        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
+        /// <returns>The resource name of the created mutate job.</returns>
+        private static string CreateMutateJob(MutateJobServiceClient mutateJobService,
+            long customerId)
+        {
+            string mutateJobResourceName =
+                mutateJobService.CreateMutateJob(customerId.ToString()).ResourceName;
+            Console.WriteLine($"Created a mutate job with resource name: " +
+                $"'{mutateJobResourceName}'.");
+
+            return mutateJobResourceName;
+        }
+
+        /// <summary>
+        /// Adds all mutate job operations to the mutate job. As this is the first time for this
+        /// mutate job, the sequence token is not set.The response will contain the next sequence
```

Comment:
Super nit: Need space between the period and 'The'

Reply:
Done